app.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+const user = require('./user')
+const connection = require('./connection')
+
+app.use(bodyParser.json({ limit: '5mb' }))
+app.use(bodyParser.urlencoded({ limit: '5mb', extended: false }))
+app.post('/item/:userName/:listName', (req, res) => {
```

Comment:
move routes definition into separate modules, outside of app.js, which you should keep tiny a free of any logic.
like:
```
app.use('/', require('./routes'))
```
in `./routes/index.js`
```
route.use('/item', require('./item.js'))
```